From 0961a383a2409c627d5437aed1f71461563567f5 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Fri, 18 Aug 2017 18:39:49 -0500
Subject: [PATCH 59/64] uhd: Add RX1 as default option

GRC bindings for the USRP sink and source suggest some antenna names.
This adds the antenna name RX1 (TwinRX) into the list of antenna name
suggestions.
---
 gr-uhd/grc/gen_uhd_usrp_blocks.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/gr-uhd/grc/gen_uhd_usrp_blocks.py b/gr-uhd/grc/gen_uhd_usrp_blocks.py
index e99de0d7d0..8b7c58682a 100644
--- a/gr-uhd/grc/gen_uhd_usrp_blocks.py
+++ b/gr-uhd/grc/gen_uhd_usrp_blocks.py
@@ -517,6 +517,10 @@ PARAMS_TMPL = """	<param>
 			<name>RX2</name>
 			<key>RX2</key>
 		</option>
+		<option>
+			<name>RX1</name>
+			<key>RX1</key>
+		</option>
 #end if
 		<tab>RF Options</tab>
 	</param>
-- 
2.11.0

